### DISCUSSION TITLE
Does Firefly III have a report that looks like YNAB's Income vs Expense table report?

### DISCUSSION THREAD

**Discussion by mumbledenoise** [firefly-iii/firefly-iii#6886] [General]
Looking for something that looks like this:

![image](https://user-images.githubusercontent.com/8943219/212887323-2630dc8f-7a8d-4380-833a-15c9ce2b5d6f.png)

Specifically:

1. Shows the month by month breakdown of categories within the Income and Expense sections, even when running the report for multiple years.
2. Shows the Income, Expense, and Net Income all in one column.

I've seen some reports which get close, the Categories (Expenses) and Categories (Revenue / income) sections of this chart in particular:

![image](https://docs.firefly-iii.org/firefly-iii/advanced-concepts/images/reports-default.png)

But this chart was generated in version 4.6.13, and from looking at the demo site and my own site, it seems these sections of the "Default financial report" have changed significantly. Am I missing it?

**Comment by JC5** [comment]
Select a larger time range if you want this report :)

**Reply by mumbledenoise** [reply]
Ah! Thanks for the quick response, that works!

When I run it for multiple years it collapses and only shows me the yearly break down, (2020,2021,2022...etc) instead of showing all the 36 months between those years. Is there a way to see all of the months when doing a report for a longer timeframe?

Is there a way to add Net Income under this for each month, like in the example YNAB chart?

**Reply by JC5** [reply]
> Is there a way to see all of the months when doing a report for a longer timeframe?

No, there isn't.

> Is there a way to add Net Income under this for each month, like in the example YNAB chart?

I'll write it down for the next layout, where reports will also be improved.

**Reply by dekiesel** [reply]
This would be great to have!
